SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -33,7 +33,9 @@ EPushRPCResult SpatialRPCService::PushRPC(Worker_EntityId EntityId, ERPCType Typ
 
 	if (EventTracer != nullptr)
 	{
-		TOptional<Trace_SpanId> SpanId = EventTracer->CreateSpan();
+		TOptional<Trace_SpanId> CauseSpanId = EventTracer->SpanIdStack.GetSpanId();
```

Comment:
Would this automatically make it so RPCs sent as a result of receiving an RPC would continue that trace chain?